test/test.js

```diff
@@ -249,6 +249,24 @@ describe('Reporting errors', function() {
           expect(err.message).to.equal('503 http response on stackdriver report');
         });
       });
+
+      it('should not handle quota exceeded error responses', function() {
```

Comment:
The test added just checks the custom logic is triggered, and therefore an `Error` is not being created. If you'd prefer an async test that actually checks the Error isn't rethrown, LMK, i can add that.